Include/XA/kzny/kewr.sid

```diff
@@ -1,122 +1,126 @@
-//KEWR SIDs INFORMATION FROM AIRAC 2006
-
-KEWR;22L:22R;LIBERTY;FIGON;FIGON;
-HUO;HUO;<br>;
-NEION;NEION;<br>;
-COATE;COATE;<br>;
-ELIOT;ELIOT;<br>;
-ETX;ETX;<br>;
-NEWEL;NEWEL;<br>;
-ZIMMZ;ZIMMZ;<br>;
-BWZ;BWZ;<br>;
-PARKE;PARKE;<br>;
-BIGGY;BIGGY;<br>;
-PTW;PTW;<br>;
-SBJ;SBJ;<br>;
-SAX;SAX;<br>;
-GAYEL;GAYEL;<br>;
-HAAYS;HAAYS;<br>;
-BREZY;BREZY;<br>;
-ELVAE;ELVAE;<br>;
-WHITE;WHITE;<br>;
-DIXIE;DIXIE;<br>;
-COL;COL;<br>;
-WAVEY;WAVEY;<br>;
-JFK;JFK;<br>;
-DPK;DPK;<br>;
-LGA;LGA;<br>;
-BDR;BDR;<br>;
-BAYYS;BAYYS;<br>;
-MERIT;MERIT;<br>;
-CMK;CMK;<br>;
-GREKI;GREKI;<br>;
-IGN;IGN;<br>;
-KEWR;22L:22R;TURNPIKE CLIMB;N040.37.17.000;W074.11.40.000;1;
-//HDG 219 TO 500  THEN HDG 215
-N040.40.09.000;W074.10.54.000;<br>;
-N040.38.34.000;W074.11.54.000;^500;
-N040.35.37.000;W074.13.25.000;
-KEWR;22L:22R;PARKWAY CLIMB;N040.36.56.000;W074.12.42.000;1;
-//RT HDG 239
-N040.40.09.000;W074.10.54.000;
-N040.38.34.000;W074.11.54.000;
-N040.36.22.000;W074.14.27.000;
-KEWR;22L:22R;BUD CLIMB;N040.38.40.000;W074.15.18.000;1;
-//RT HDG 263
-N040.40.09.000;W074.10.54.000;
-N040.38.34.000;W074.11.54.000;
-N040.37.34.000;W074.16.06.000;
+KEWR;22L:22R:29;LIBERTY;WUDGO;WUDGO;
+40.69083154;-74.18138771;
+HUO;HUO;
+40.69083154;-74.18138771;<br>;
+NEION;NEION;
+40.69083154;-74.18138771;<br>;
+COATE;COATE;
+40.69083154;-74.18138771;<br>;
+ELIOT;ELIOT;
+40.69083154;-74.18138771;<br>;
+ETX;ETX;
+40.69083154;-74.18138771;<br>;
+NEWEL;NEWEL;
+40.69083154;-74.18138771;<br>;
+ZIMMZ;ZIMMZ;
+40.69083154;-74.18138771;<br>;
+BWZ;BWZ;
+40.69083154;-74.18138771;<br>;
+PARKE;PARKE;
+40.69083154;-74.18138771;<br>;
+BIGGY;BIGGY;
+40.69083154;-74.18138771;<br>;
+PTW;PTW;
+40.69083154;-74.18138771;<br>;
+SBJ;SBJ;
+40.69083154;-74.18138771;<br>;
+SAX;SAX;
+40.69083154;-74.18138771;<br>;
+GAYEL;GAYEL;
+40.69083154;-74.18138771;<br>;
+HAAYS;HAAYS;
+40.69083154;-74.18138771;<br>;
+BREZY;BREZY;
+40.69083154;-74.18138771;<br>;
+ELVAE;ELVAE;
+40.69083154;-74.18138771;<br>;
+WHITE;WHITE;
+40.69083154;-74.18138771;<br>;
+DIXIE;DIXIE;
+40.69083154;-74.18138771;<br>;
+COL;COL;
+40.69083154;-74.18138771;<br>;
+WAVEY;WAVEY;
+40.69083154;-74.18138771;<br>;
+BDR;BDR;
+40.69083154;-74.18138771;<br>;
+BAYYS;BAYYS;
+40.69083154;-74.18138771;<br>;
+MERIT;MERIT;
+40.69083154;-74.18138771;<br>;
+CMK;CMK;
+40.69083154;-74.18138771;<br>;
+GREKI;GREKI;
+40.69083154;-74.18138771;<br>;
+IGN;IGN;
 
 KEWR;04L:04R:11:22L:22R:29;NEWARK;WUDGO;WUDGO;

```

Comment:
These two SIDs (NEWARK & LIBERTY) seem to be identical except that the NEWARK SID allows more runways. Not sure if this is intentional or an import error. I've tried checking the FAA publications of the SIDs, but the diagrams are too chaotic for me to correlate with Aurora's view. I'll approve it assuming the difference between the SIDs is not related to the routing as displayed on Aurora.

Reply:
Indeed. One SID is only for departures out of 22L/22R while the other is for all runways. It is a vectors only SID but I included it show all accepted exits.